ext/nnstreamer/tensor_filter/tensor_filter_python_api.c

```diff
@@ -0,0 +1,217 @@
+/**
+ * GStreamer Tensor_Filter, Python Module
+ * Copyright (C) 2019 Dongju Chae <dongju.chae@samsung.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ */
+/**
+ * @file	tensor_filter_python_api.c
+ * @date	10 Apr 2019
+ * @brief	python helper structure for nnstreamer tensor_filter
+ * @see		http://github.com/nnsuite/nnstreamer
+ * @author	Dongju Chae <dongju.chae@samsung.com>
+ * @bug		No known bugs except for NYI items
+ */
+
+#include "tensor_filter_python_core.h"
+
+/** @brief object structure for custom Python type: TensorShape */
+typedef struct {
+  PyObject_HEAD
+  PyObject *dims;
+  PyArray_Descr* type;
+} TensorShapeObject;
+
+/** 
+ * @brief method impl. for setDims
+ * @param self : Python type object 
+ * @param args : arguments for the method
+ */
+static PyObject *
+TensorShape_setDims(TensorShapeObject *self, PyObject *args) {
+  PyObject *dims = args;
+  PyObject *new_dims;
+
+  // PyArg_ParseTuple() returns borrowed references
```

Comment:
Don't use C99 style comments with *.c files.

Reply:
Got it